classes/submission/form/SubmissionSubmitStep1Form.inc.php

```diff
@@ -105,4 +105,28 @@ function setSubmissionData($submission) {
 		$submission->setSeriesPosition($this->getData('seriesPosition'));
 		parent::setSubmissionData($submission);
 	}
+
+	/**
+	 * Save changes to submission.
+	 * @return int the submission ID
+	 */
+	function execute(...$functionParams) {
+		
+		$submissionId = parent::execute(...$functionParams);
+
+		$request = Application::get()->getRequest();
+
+		$seriesDao = \DAORegistry::getDAO('SeriesDAO'); /** @var SeriesDAO $seriesDao */
+
+        $series = $seriesDao->getById((int)$request->getUserVar('seriesId'), (int)$request->getContext()->getId());
```

Comment:
Looks like some wobbly indentation here too. The `stable-3_3_0` branch doesn't use the code formatter; we're indenting there using tab characters. We've only adopted PSR2 starting with 3.4.0 (the `main` branch).

Reply:
fixed.